js/app.js

```diff
@@ -1,42 +1,123 @@
-const pokemonList = document.getElementById('pokemon-list');
-const loadMoreBtn = document.getElementById('load-more');
+const pokemonList = document.querySelector('.pokemon-list');
+const loadMoreBtn = document.querySelector('.load-more-button');
 
+const pokemons = [];
 let offset = 0;
 const limit = 12;
```

Comment:
```suggestion
const pokemons = [];
const limit = 12;
let offset = 0;
```

Групуй `const` до `const`, `let` до `let` - так гарніше і легше читається